core/src/main/java/com/scalar/db/util/ActiveExpiringMap.java

```diff
@@ -183,27 +193,25 @@ public int size() {
   }
 
   public void updateExpirationTime(K key) {
-    ValueHolder<V> value = map.get(key);
+    ValueHolder value = map.get(key);
     if (value != null) {
       value.updateExpirationTime();
     }
   }
 
   @VisibleForTesting
   @Nullable
-  ValueHolder<V> getValueHolder(K key) {
+  ValueHolder getValueHolder(K key) {
     return map.get(key);
   }
 
   @VisibleForTesting
-  static class ValueHolder<V> {
+  class ValueHolder {
```

Comment:
Made it non-static class so that this class can use the field of the outer class `valueLifetimeMillis`.